packages/cactus-plugin-ledger-connector-corda/src/main-server/kotlin/gen/kotlin-spring/build.gradle.kts

```diff
@@ -5,9 +5,10 @@ version = project.properties["version"]!!
 
 val corda_release_group = "net.corda"
 val corda_core_release_group =  "net.corda"
-val corda_release_version = "4.6"
-val corda_core_release_version = "4.6"
-val corda_platform_version = 5
+val corda_release_version = "4.12-HC02"
```

Comment:
Why `4.12-HC02`, that's 7 months older? Why not stable `4.12`?

Reply:
@sandeepnRES Thanks for catching that! This PR has been in the works for a very long time (about 7 months) so back then there was no 4.12 GA but now I agree it should be upgraded. It's not as trivial unfortunately as setting the dependency version here and be done with it because I also have to upgrade the kotlin samples, but it's important enough that I'll just do it now rather than have to come back to it later!

Reply:
@sandeepnRES Took me all day in the end but I made the changes and also a few bug-fixes along the way so now it's much more stable than it was, thank you very much once again for noticing this!

Reply:
Thanks @petermetz. I will merge this now since all the tests are passing.